source/style-guide/information-types.txt

```diff
@@ -0,0 +1,75 @@
+.. _information-types:
+
+=================
+Information Types
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/info-typing/fact-general-guidance.rst
+
+The following sections describe information typing and provide
+prototypes that show how to apply this strategy to our documentation.
+
+What is Information Typing?
+---------------------------
+
+Information typing categorizes content by type based on your audience's
+goals. Throughout the writing process, information typing helps keep the
+information that you write focused on those goals.
+
+Additionally, information typing accomplishes the following goals:
+
+- Separate content into categories and delivers roughly one type at a
+  time.
+
+- Account for the purpose of a page before you write it.
+
+- Answer the user question "why do I care?" or "what does this mean for
+  me?" quickly.
+
+- Use a suggested format to help you get started on a blank page.
+
+What Information Types Do We Use?
+---------------------------------
+
+Currently, some teams use the following information types:
+
+Concept
+  Helps a user learn about a concept or new feature. Concept pages
+  intend to answer the following questions:
+  
+  - "What is this feature?"
+  
+  - "Why do I care about this feature?"
+  
+  - "How does this feature work?"
+  
+  The audience includes users who have never used a particular feature
+  or command. 
+
+Task
+  Helps a user do a job. The audience includes users who want to
+  accomplish a particular goal. They might approach this task for the
+  very first time, have limited experience, or need a refresher. This
+  page should contain all the necessary information for someone to
+  complete the task, any necessary conceptual information,
+  prerequisites, steps, and expected results. 
+
+Reference
+  Delivers granular details about a thing. The audience includes
+  experienced users. Apart from straightforward reference manual pages,
+  reference pages can include bulleted lists or tables of information. 
+
```

Comment:
@sarah-olson-mongodb curious for your thoughts on this. What do you think about adding a new content type for Glossary items? It could be worthwhile because glossary entries don't really fit into any of the other content types. If we want to add a new content type, I can create a follow-up ticket.

Reply:
Hey @jeff-allen-mongo, @ccho-mongodb. Good question, and thanks for raising it. 

The organizing principle for information typing is user goals. The idea is that understanding what a type of user is wanting to accomplish helps us address that goal consistently, throughout the page we are writing. Based on this, I believe we can wrap Glossary entries into an option for the Reference type. 

I think we could really benefit from adding some alternatives to our current Reference prototype, however, and Glossary pages could be one of those.

Reply:
That makes sense to me. I created https://jira.mongodb.org/browse/DOCSP-32981 to track those updates in a separate ticket.

Reply:
Since the glossary type is standardized in DITA and classifying it as a Reference type would require creating an alternative prototype page for the format, I think it could make more sense to identify it as a separate information type.  While I think it would help the organization of the content, I'll leave it up to your preference.

Reply:
I'm a little torn and I think it may be easier to decide on a direction after seeing the content staged. I updated the description of [DOCSP-32981](https://jira.mongodb.org/browse/DOCSP-32981) to indicate that we may want to try making Glossary entries their own type.